image-sync/oc-mirror/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM mcr.microsoft.com/cbl-mariner/base/core:2.0 AS downloader
+FROM --platform=linux/amd64 mcr.microsoft.com/cbl-mariner/base/core:2.0 AS downloader
```

Comment:
Do we need to specify platform here?  Should the container infer it depending on the platform you run on?